### PR TITLE
Feat/filter list

### PR DESCRIPTION
**Pull request by alvaroha1**
- Needs to be tested with the backend and then unify the fetch to the api.
- Also i'm planning to add _debounce.

### CODE REVIEW

File: src/components/Filter.js [hunk 1]
```diff
@@ -1,56 +1,73 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { debounce } from 'lodash'
+import Slider, { Range } from 'rc-slider'
```
Comment (ps3611) [inline src/components/Filter.js]: the ';' are missing no? does your linter not give error?

File: src/components/Filter.js [hunk 2]
```diff
@@ -1,56 +1,73 @@
...
-			size:'',
+			price:1000 * 1000,
+			size:100,
 		}
 	}
 
-	handleSlidersChange = (e) => {
-		console.log(e);
-		// const target = event.target;
-    // const value = target.value;
```
Comment (ps3611) [inline src/components/Filter.js]: remove comments

File: src/components/Filter.js [hunk 3]
```diff
@@ -1,56 +1,73 @@
...
-						</div>
-					</div>
-				</div>
 
-				<div>
-					<h4><strong>Price</strong></h4>
-					<input className="slider is-fullwidth is-medium" step="1" min="0" max="100" value="50" type="range" onChange={this.handleInputChange}></input>
+				<h4><strong>Max Price</strong></h4>
+				<div className="slider">
+					<Slider min={0} max={2000} marks={price} included={false} defaultValue={1000} onChange={this.setPrice}/>
```
Comment (ps3611) [inline src/components/Filter.js]: maybe cleaner to have this on multiple lines..

File: src/components/Filter.js (hunk 1 above)
Comment (alvaroha1) [reply src/components/Filter.js]: It was complaining before when I wrote them but not anymore. I will add them now.

File: src/components/Filter.js [hunk 4]
```diff
@@ -1,56 +1,73 @@
...
 	}
 }
 
-export default Filter
\ No newline at end of file
+const mapDispatchToProps = (dispatch) => ({
+	filterHomes: () => dispatch({
+		type: 'FILTER_HOMES',
+		method: 'POST',
+    api: {
```
Comment (ps3611) [inline src/components/Filter.js]: indentation nightmare! 
are you using the linter?

File: src/redux/reducers.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 const initalState = {
 	isMapOn: false,
-	homes: {}
+	homes: {},
+	fHomes: {},
```
Comment (ps3611) [inline src/redux/reducers.js]: fHomes is not good naming!!

File: src/components/Filter.js [hunk 6]
```diff
@@ -1,56 +1,73 @@
...
+	}
+
+	setSize = (value) => {
+		this.setState({size: value})
+		this.getQuery();
+	}
+
+	getQuery = () => {
+		const filter = qs.stringify(this.state)
+		console.log(filter);
```
Comment (ps3611) [inline src/components/Filter.js]: no console log please!

File: src/components/Filter.js [hunk 7]
```diff
@@ -1,56 +1,73 @@
...
+import 'rc-slider/assets/index.css'
+import './Filter.css'
+import qs from 'qs'
+
+const size = {
+	0: <strong>0</strong>,
+	200: <strong>200m</strong>,
+}
+
+const price = {
```
Comment (ps3611) [inline src/components/Filter.js]: better naming please!
price what?
priceFilterLabel maybe better

File: src/components/Filter.js (hunk 4 above)
Comment (alvaroha1) [reply src/components/Filter.js]: Yes it doesnt look like this in my screen. Might be a windows thing i'll look it up.

**Pull request merged**